operations/CSM_product_management/Configure_Compute_Nodes_with_CFS.md

```diff
@@ -0,0 +1,59 @@
+# Configure Compute Nodes with CFS
+
+CSM includes a playbook that should be applied to any Compute node images.
+The `compute_nodes.yml` playbook installs the packages for both the CFS and BOS reporters.
+These packages are necessary for CFS and BOS to run, so a configuration layer containing the
+playbook must be included in the image customization for any nodes that are expected to be
+managed with CFS and BOS.
+
+## Setting up the CSM Configuration layer
+
+To setup the compute configuration layer, first gather the following information:
+
+* HTTP clone URL for the configuration repository in [VCS](../configuration_management/Version_Control_Service_VCS.md)
+* Path to the Ansible play to run in the repository
+* Commit ID in the repository for CFS to pull and run on the nodes.
+
+| Field | Value  | Description  |
+|:----------|:----------|:----------|
+| cloneUrl | `https://api-gw-service-nmn.local/vcs/cray/csm-config-management.git` | CSM configuration repo |
+| commit  | **Example:** `5081c1ecea56002df41218ee39f6030c3eebdf27` | CSM configuration commit hash |
+| name | **Example:** `csm-<version>` | CSM Configuration layer name |
+| playbook | `compute_nodes.yml` | Default Ansible playbook for CSM configuration of compute nodes |
+
+1. Retrieve the commit in the repository to use for configuration. If changes
+   have been made to the default branch that was imported during a CSM
+   installation or upgrade, use the commit containing the changes.
+
+1. If no changes have been made, the latest commit on the default branch for
+   this version of CSM should be used. Find the commit in the
+   `cray-product-catalog` for the current version of CSM. For example:
+
+   ```bash
+   ncn# kubectl -n services get cm cray-product-catalog -o jsonpath='{.data.csm}'
+
+   1.2.0:
+      configuration:
+         clone_url: https://vcs.cmn.SYSTEM_DOMAIN_NAME/vcs/cray/csm-config-management.git
+         commit: 43ecfa8236bed625b54325ebb70916f55884b3a4
+         import_branch: cray/csm/1.9.24
+         import_date: 2021-07-28 03:26:01.869501
+         ssh_url: git@vcs.cmn.SYSTEM_DOMAIN_NAME:cray/csm-config-management.git
+      ...
+   ```
+
+   The commit will be different for each system and version of CSM. For
+   this example, it is:
+
+         43ecfa8236bed625b54325ebb70916f55884b3a4
+
+1. Craft a new configuration layer entry for CSM using the procedure in [Update a CFS Configuration](../configuration_management/Update_a_CFS_Configuration.md):
+
+  ```json
```

Comment:
```suggestion
  The following is an example entry for the JSON configuration file:

  ```json
```
It might be good to add some kind of statement here to add more context before an admin moves to the "Update a CFS Configuration" procedure.